src/mbgl/gl/attribute.hpp

```diff
@@ -27,10 +25,8 @@ class AttributeBinding {
           type(DataTypeOf<T>::value),
           count(N),
           offset(O) {
-        static_assert(std::is_standard_layout<Vertex>::value, "vertex type must use standard layout");
```

Comment:
This one moved to [the `Drawable` constructor](https://github.com/mapbox/mapbox-gl-native/blob/ca28e8319522829dc9315f4c94d92d5a759ceb9c/src/mbgl/gl/drawable.hpp#L105). I just dropped "vertex type is too big", because it was kind of silly.